fission/src/ui/modals/spawning/ManageAssembliesModal.tsx

```diff
@@ -1,30 +1,50 @@
-import React, { useReducer } from "react"
+import React, { useMemo, useReducer } from "react"
 import Modal, { ModalPropsImpl } from "@/components/Modal"
 import { FaWrench } from "react-icons/fa6"
 import Button from "@/components/Button"
 import Label, { LabelSize } from "@/components/Label"
 import World from "@/systems/World"
 import MirabufSceneObject from "@/mirabuf/MirabufSceneObject"
+import { usePanelControlContext } from "@/ui/PanelContext"
+import { setSelectedBrainIndexGlobal } from "@/ui/panels/configuring/ChooseInputSchemePanel"
+import SynthesisBrain from "@/systems/simulation/synthesis_brain/SynthesisBrain"
+import { useModalControlContext } from "@/ui/ModalContext"
+import InputSystem from "@/systems/input/InputSystem"
 
 interface AssemblyCardProps {
     mira: MirabufSceneObject
     update: React.DispatchWithoutAction
 }
 
 const AssemblyCard: React.FC<AssemblyCardProps> = ({ mira, update }) => {
+    const { openPanel } = usePanelControlContext()
+    const { closeModal } = useModalControlContext()
+
+    const brain = useMemo(() => (mira.brain as SynthesisBrain)?.brainIndex, [mira])
+
     return (
         <div
             key={mira.id}
             className="flex flex-row align-middle justify-between items-center bg-background rounded-sm p-2 gap-2"
         >
-            <Label className="text-wrap break-all">{mira.assemblyName}</Label>
-            <Button
-                value="Delete"
-                onClick={() => {
-                    World.SceneRenderer.RemoveSceneObject(mira.id)
-                    update()
-                }}
-            />
+            <Label className="text-wrap break-all">{`[${InputSystem.brainIndexSchemeMap.get(brain)?.schemeName ?? "-"}] ${mira.assemblyName}`}</Label>
+            <div className="flex flex-row gap-4">
+                <Button
+                    value="Set Scheme"
```

Comment:
```suggestion
                    value="Edit Scheme"
```

Reply:
This doesn't edit the scheme, it sets it